src/main/scala-2.11/markets/clearing/ClearingMechanismActor.scala

```diff
@@ -16,10 +16,11 @@ limitations under the License.
 package markets.clearing
 
 import akka.actor.{Props, ActorRef}
+import akka.agent.Agent
 
-import markets.{Canceled, Cancel, BaseActor}
+import markets.tickers.Tick
+import markets.BaseActor
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1560525558/issues/source?bid=2921361&fileBranchId=3019109#l22)